pkgs/development/python-modules/rasterio/default.nix

```diff
@@ -7,14 +7,14 @@
 
 buildPythonPackage rec {
   pname = "rasterio";
-  version = "1.2.6";
+  version = "4";
```

Comment:
```suggestion
  version = "1.3a3";
```

Reply:
I have a fixup queued locally.

Reply:
I pushed an update to 1.2.10 that I tested on master.